src/main/kotlin/com/skaggsm/treechoppermod/FabricTreeChopper.kt

```diff
@@ -17,8 +18,8 @@ object FabricTreeChopper : ModInitializer {
     const val MODID = "fabric-tree-chopper"
 
     val serializer = JanksonValueSerializer(false)
-    val configFile = Path.of("config","fabric-tree-chopper.json")
-    val oldConfigFile = Path.of("config", "fabric-tree-chopper.toml")
+    val configFile = Paths.get("config/fabric-tree-chopper.json")
+    val oldConfigFile = Paths.get("config/fabric-tree-chopper.toml")
```

Comment:
Using `Paths.get` is fine, but you shouldn't use the `/` file separator in strings because it's not always cross-compatible from *nix environments to Windows environments. The original parameters from `Path.of` should work the same in `Paths.get`.